File: Cap 04/Topico 4.5/Topico 4.5.1/Decisao Sequencial/decseq.go
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Topico 4.5.1
 * 
 * Programa ..: OPERACAO COM DECISAO SEQUENCIAL
 */

package main

import (
  "fmt"
)

var N int

func main() {
	
  fmt.Println("DECISAO SEQUENCIAL")
  fmt.Println("")
  
  fmt.Print("Entre valor inteiro <N>: ")
  fmt.Scan(&N)
  
  if (N == 1) {
    fmt.Println("voce entrou o valor 1")
  }
  
  if (N == 2) {
    fmt.Println("voce entrou o valor 2")
  }
  
  if (N < 1) {
    fmt.Println("voce entrou valor muito baixo")
  }
  
  if (N > 2) {
    fmt.Println("voce entrou valor muito alto")
  }
  
}

File: Cap 04/Topico 4.5/Topico 4.5.1/Decisao Sequencial/decseq.pl
=pod

 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Topico 4.5.1
 * 
 * Programa ..: OPERACAO COM DECISAO SEQUENCIAL
 
=cut

use strict;
use warnings;

my $N;

print "DECISAO SEQUENCIAL\n\n";

print "Entre valor inteiro <N>: ";
$N = <STDIN>;

if ($N == 1) {
    print "voce entrou o valor 1\n";
}

if ($N == 2) {
    print "voce entrou o valor 2\n";
}

if ($N < 1) {
    print "voce entrou valor muito baixo\n";
}

if ($N > 2) {
    print "voce entrou valor muito alto\n";
}

exit 0;
